src/vm/prestub.cpp

```diff
@@ -49,6 +49,10 @@
 #include "stacksampler.h"
 #endif
 
+#ifdef FEATURE_PERFMAP
```

Comment:
You have some duplicate ifdeffing around here.  Because perfmap.h won't have anything in it unless FEATURE_PERFMAP is specified, this ifdef seems unecessary